tests/GeoNames/ClientTest.php

```diff
@@ -354,4 +354,14 @@ public function testConnectTimeout()
 
         $this->assertEquals(10, $g->getConnectTimeout());
     }
+
+    public function testOverrideApiUrl()
+    {
+        $client = new GeoNamesClient(
+            $this->config['username'],
+            $this->config['token'],
+            ['apiUrl' => 'http://api.geonames.org']
+        );
```

Comment:
Might it be a good idea to add the extra param to the client instance created in the `setUp()` method and then using that in the test? It would make the new test consistent with the other existing ones.